### PR TITLE
MGMT-16276: Test OCP 4.15 with RHEL 9.4

### PR DESCRIPTION
**Pull request by danmanor**

### CODE REVIEW

**Comment by danmanor** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-4-15-rhel-9-4

File: ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml [hunk 1]
```diff
@@ -970,6 +970,14 @@ tests:
       ASSISTED_CONFIG: |
         OCI_BOOT_VOLUME_TYPE="ISCSI"
     workflow: assisted-oci
+- as: e2e-metal-assisted-4-15-rhel-9-4
```
Comment (adriengentil) [inline ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml:973]:
```suggestion
- as: e2e-metal-assisted-4-15-rhel-9-4
  always_run: false
```

**Comment by danmanor** [comment]
AFAIK only when there will be stable releases no?

**Pull request closed without merging**